problems/1.types+operators.js

```diff
@@ -2,19 +2,34 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if (val === val) {
```

Comment:
val will always be equal to value. 
you want to be checking if val is of type number. 
typeof val === "number"